source/draw/Image.ooc

```diff
@@ -96,7 +96,7 @@ Image: abstract class {
 		fontSize := DrawContext getFontSize(fontAtlas)
 		viewport := IntBox2D new(localOrigin, fontSize)
 		targetOffset := IntPoint2D new(0, 0)
-		characterDrawState := DrawState new(this) setInputImage(fontAtlas) setBlendMode(BlendMode Add)
+		characterDrawState := DrawState new(this) setInputImage(fontAtlas) setBlendMode(BlendMode White)
```

Comment:
The blend mode for text is now called white because we have real additive blending available.